README.md

```diff
@@ -203,10 +203,14 @@ The pattern should something like "base-%03d.html"
 ### hocr-wordfreq
 
 ```
-hocr-wordfreq file.html
+hocr-wordfreq [-h] [-i] [-n MAX] [file.html]
```

Comment:
We don't mention the `-h` option in any of the other commands, also it is always there.